src/components/input/Input.vue

```diff
@@ -101,8 +107,22 @@ export default {
                 return this.newValue
             },
             set(value) {
-                this.newValue = value
-                this.$emit('input', value)
+                /*If forcedSync - set localValue same as emited instantly(not default behaviour, for example v-model.trim changing value only on blur*/
+                if(this.forcedSync){
+                    if(this.trim)value=value.trim();
+                    else if(this.number)value = this.toNumber(value);
+                    //add your handlers and preparers here, for forced sync on conditional setters/getters inside parent computed/event's logic
+					// if(this.onlyIntegers){//allow input only integers
+						// if(!Number.isInteger(value))return;
+					// }
+					
+                }
+                if(this.prepare){//not related to v-model stuff
```

Comment:
Not sure about this one, is this real needed?
Coz if we gonna update local value inside local setter with changes it's gonna be reactive too, but if it's focused right now, its not gonna be updated instantly.